src/filesystem.rs

```diff
@@ -26,12 +26,35 @@ fn paths_from_path_param(env_var: &str) -> impl Iterator<Item = &str> {
     env_var.split(':').filter(|folder| folder != &"")
 }
 
+fn compiled_default_path(path: Option<&str>) -> Option<PathBuf> {
+    match path {
+        Some(path) => {
+            let path = if path.contains(MAIN_SEPARATOR) {
+                path.split(MAIN_SEPARATOR).next().unwrap()
```

Comment:
What is this supposed to do? It doesn’t work correctly, `"/etc/xdg/navi/config.yaml"` is transformed to `""`.